feature_extractor.py

```diff
@@ -261,10 +262,12 @@ def polar_to_cartesian(coordinates):
     print(x.shape, x.dtype)
     print(y.shape, y.dtype)
     '''
-    
-    extract_seldnet_data('/media/data1/datasets/DCASE2020/foa_dev',
-                         'mic_dev',
-                         '/media/data1/datasets/DCASE2020/metadata_dev',
-                         'mic_label',
-                         mode='mic')
+    path = '/media/data1/datasets'
+    if not os.path.exists(path):
+        path = '/root/datasets'
+    extract_seldnet_data(path + '/DCASE2020/foa_dev',
```

Comment:
Because, directly adding paths are not recommended, how about using os.path.join?